src/main/java/nextstep/fp/StreamStudy.java

```diff
@@ -27,7 +27,9 @@ public static void printLongestWordTop100() throws IOException {
                 .get("src/main/resources/fp/war-and-peace.txt")), StandardCharsets.UTF_8);
         List<String> words = Arrays.asList(contents.split("[\\P{L}]+"));
 
-        // TODO 이 부분에 구현한다.
+        words.stream().filter(x -> x.length() > 12).distinct()
+            .sorted().limit(100)
+            .forEach(x -> System.out.println(x.toLowerCase()));
```

Comment:
적절한 개행을 활용해서 stream의 가독성을 높여봐도 좋을 것 같아요 😃 
```suggestion
        words.stream()
            .filter(x -> x.length() > 12)
            .distinct()
            .sorted()
            .limit(100)
            .forEach(x -> System.out.println(x.toLowerCase()));
```